fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -925,33 +925,11 @@ bool parseNetWorkAddrFromKeys(ReadYourWritesTransaction* ryw,
 }
 
 ACTOR Future<bool> checkExclusion(Database db,
-                                  std::vector<AddressExclusion>* addresses,
-                                  std::set<AddressExclusion>* exclusions,
+                                  const std::vector<AddressExclusion>* addresses,
+                                  const std::set<AddressExclusion>* exclusions,
+                                  const std::unordered_set<std::string>* localities,
```

Comment:
`const *` doesn't prevent modifying the container's content. we want for instance, `const std::set<AddressExclusion>& exclusions`.